go.mod

```diff
@@ -89,6 +89,7 @@ require (
 	golang.org/x/oauth2 v0.0.0-20200902213428-5d25da1a8d43
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
 	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e
+	gomodules.xyz/jsonpatch/v2 v2.0.1
```

Comment:
Using this because it's already in go.sum, but k8s mentions using https://github.com/gomodules/jsonpatch internally (https://github.com/kubernetes-sigs/controller-runtime/issues/281)